node/heartbeat/sender.go

```diff
@@ -104,3 +119,18 @@ func (s *Sender) SendHeartbeat() error {
 
 	return nil
 }
+
+func (s *Sender) updateMetrics(hb *Heartbeat) {
+	result := s.computePeerList(hb.Pubkey)
+	s.statusHandler.SetStringValue(core.MetricPeerType, result)
+}
+
+func (s *Sender) computePeerList(pubkey []byte) string {
+	peerType, _, err := s.peerTypeProvider.ComputeForPubKey(pubkey)
+	if err != nil {
+		log.Warn("monitor: compute peer type", "error", err)
```

Comment:
warn message is incorrect, this is not in monitor, this is in sender

Reply:
done